pytest_sftpserver/core.py

```diff
@@ -0,0 +1,19 @@
+from pytest_sftpserver.sftp.server import SFTPServer
+
+
+class SFTPServerContextDecorator(object):
+    def __enter__(self):
+        self.server = SFTPServer()
+        self.server.start()
+        return self.server
+
+    def __exit__(self, *args, **kwargs):
+        if self.server.is_alive():
+            self.server.shutdown()
+
+    def __call__(self, func):
+        def wrapper(*args, **kwargs):
```

Comment:
It's always a nice convenience to decorate wrappers with `@wraps(func)` to preserve the function signature.